sympy/simplify/tests/test_radsimp.py

```diff
@@ -157,6 +157,34 @@ def test_radsimp_issue_3214():
     assert radsimp(b) == -I*(c + I*p - sqrt(c**2 - p**2))**2/(2*c*p)
 
 
+def test_rationalize():
+    assert rationalize(sqrt(2) - 1) == (1 + sqrt(2), 1)
+    assert rationalize(sqrt(2) + sqrt(3) + sqrt(5)) == \
+        (2*sqrt(6)*(-sqrt(5) + sqrt(2) + sqrt(3)), 24)
```

Comment:
I now get a difficult issue how the result cycles around `2*sqrt(6)*(-sqrt(5) + sqrt(2) + sqrt(3))`, `-(4 - 2*sqrt(10))*(-sqrt(3) + sqrt(2) + sqrt(5))`, `-(6 - 2*sqrt(15))*(-sqrt(2) + sqrt(3) + sqrt(5))` with different python instances. Maybe this has to do with using `set` or due to some issue in sympy.